pandas/io/parsers.py

```diff
@@ -757,6 +757,16 @@ def read_table(
     memory_map=False,
     float_precision=None,
 ):
+    # TODO: validation duplicated in read_csv
+    if delim_whitespace and (delimiter is not None or sep != "\t"):
+        raise ValueError(
+            "Specified a delimiter with both sep and "
+            "delim_whitespace=True; you can only specify one."
+        )
+    if delim_whitespace and sep == "\t":
```

Comment:
the `and sep == ...` is probably not now needed and could have just `if delim_whitespace` as an outer condition but nbd if this is a transient change

Reply:
Thanks, you are right of course. Changed it.